spec/features/html_validity_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "HTML validity" do
```

Comment:
Can this be combined with the accessibility spec? Otherwise, we're unlikely to remember to add additional to both tests.

Reply:
Good point. Maybe there's a way to have a list of pages to loop through in one place?

Reply:
@ndushay I looked into automating this and it requires going way outside RSpec best practices, and it was ugly.

@justinlittman I can add these tests to the a11y spec, sure.

Reply:
Combined these into a single file.